esx_service/cli/vmdkops_admin_sanity_test.py

```diff
@@ -37,7 +37,7 @@ def test_ls(self):
         output = subprocess.check_output(cmd, shell=True, stderr=self.devnull)
         lines = output.split('\n')
         divider_columns = lines[1].split()
-        self.assertEqual(8, len(divider_columns))
+        self.assertEqual(10, len(divider_columns))
```

Comment:
this used to be (and still is) hard to read. Can you add a variable so there is a name involved? Something like 
`expected_column_count = 10`